lib/options.nix

```diff
@@ -184,18 +184,23 @@ rec {
     if length defs == 1
     then (head defs).value
     else assert length defs > 1;
-      throw "The option `${showOption loc}' is defined multiple times.\n${message}\nDefinition values:${showDefs defs}";
+      throw "The option `${showOption loc}' is defined multiple times while it's expected to be unique.\n${message}\nDefinition values:${showDefs defs}";
 
   /* "Merge" option definitions by checking that they all have the same value. */
   mergeEqualOption = loc: defs:
+    let
+      suggestion = ''
+        Use `lib.mkForce value` or `lib.mkDefault value` to change the priority on any of these definitions.
```

Comment:
It wouldn't be clear to me what mkDefault does. Might be good to refer to the manual, though the manual also doesn't really explain this. Maybe add a link and then the manual could be fixed in a future PR?

https://nixos.org/manual/nixos/stable/#sec-option-definitions-setting-priorities